hibike/hibike_message.py

```diff
@@ -104,6 +106,14 @@ def __str__(self):
     def __repr__(self):
         return str(self)
 
+    # pylint: disable=protected-access
+    def __eq__(self, other):
+        if not isinstance(other, HibikeMessage):
+            return False
+        return self._message_id == other._message_id and\
```

Comment:
For multi line, preferred style is implicit continuation via wrapping in parentheses, and putting the boolean operator at the start of the line. 

see [here](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator)